resources/test/fixtures/SPR001.py

```diff
@@ -15,7 +15,7 @@ def method(self):
 
     def wrong(self):
         parent = super(Child, self)  # wrong
-        super(Child, self).method  # wrong
+        super(Child, self).method()  # wrong
```

Comment:
This was on purpose: `ast.Attribute` vs `ast.Call`

Reply:
Oh right, will revert. (The code works fine w/ either, didn't realize it was intentional but makes total sense.)